docs/schema/snitch_cluster.schema.json

```diff
@@ -422,6 +422,12 @@
                                     "description": "Enable Snitch Accelerator eXtension (SNAX)",
                                     "default": false
                                 },
+                                "snax_tcdm_ports": {
+                                    "type": "number",
+                                    "title": "SNAX TCDM Port",
+                                    "description": "Number of ports dedicated for an accelerator",
```

Comment:
Does this require SNAX to be enabled? Maybe mention this?

Reply:
Hmm good question. Yes, SNAX needs to be enabled. But it should be fine since these just gets into the parameters but they become unused.

Reply:
Actually, SNAX does NOT need to be enabled. Or rather, if SNAX is enabled it uses this parameter. Otherwise it will just ignore it.